src/main/java/ro/amihaescu/springreact/controller/BeerController.java

```diff
@@ -24,4 +25,17 @@ public BeerController(BeerRepository repository) {
         return repository.findAll();
     }
 
+    @GetMapping("/good-beers")
+    @CrossOrigin
+    public List<Beer> goodBeers(){
+        List<Beer> allBeers = repository.findAll();
+        List<Beer> goodBeers = new ArrayList<>();
+        for (Beer beer : allBeers) {
```

Comment:
This is something from 1965. Use java 8, stream 👎

Reply:
@antaka1 fixed it. Can you please re-check?

Reply:
Welcome to 2018. Approved :)